### ISSUE TITLE
suggest

### ISSUE THREAD

**Issue by yubang** [muatik/flask-profiler#42]
能否记录下返回的结果呢，因为这个插件在用于测试API接口的时候需要这个功能，这样子便于调试。

Can record the result returned, because this plug-in when used to test the API interface to need this functionality, it easy to debug.

**Comment by muatik** [comment]
Indeed, it can be a useful functionality. It would be better to have a settings directive to turn this functionality on or off because some API outputs may be too large to store.

**Comment by muatik** [comment]
@yubang would you like to participate as a coder in the implementation of this?
